### ISSUE TITLE
gulpfile.babel.js / config.yml: Adding a variable for dist/ output path

### ISSUE THREAD

**Issue by phifa** [zurb/foundation-sites#8262]
I am excited about the new config.yml file. Makes adding js plugins much easier! Thanks for the great implementation! 

Would it be possible to add a variable in PATHS for the where the dist/ folder will be created? I always did this manually, but I thought maybe this could be added as a feature, as I might not be the only person who needs this. When using foundation with a CMS or some other context, often the Public folder for assets is a given and the foundation'd dist/ setting needs to be adjusted. 

The way I did this in the past was like this (gulpfile.js): 

```
// File paths to various assets are defined here.
var PATHS = {
  dist: '../../Public/project/dist',
  assets: [
    'src/assets/**/*',
   ...
```

And then I changed all occurences of dist/ like this: 

```
// Delete the "dist" folder
// This happens every time a build starts
gulp.task('clean', function(done) {
  rimraf(PATHS.dist, done);
});
...
```

**Comment by colin-marshall** [comment]
I think this is a great idea that allows for more flexibility with the templates. I know we are planning on making the templates into Yeoman generators so this would be a good thing to include as an option when generating a Foundation Sites app, but would also be easy to add to the templates now.

**Comment by phifa** [comment]
So should I do a pull request? Never done that before but I guess I could :smile: Or maybe safer you or someone else does it? Just wanting to help.

**Comment by colin-marshall** [comment]
I can definitely do it for you, but I would recommend doing it yourself so you get credit. The first thing you should do is make a fork of the project on github and then clone the fork locally. Then follow the steps in this guide: https://guides.github.com/introduction/flow/ for the rest of the workflow.

Before you do anything though, let's wait and see what @gakimball wants to include this change so you don't end up wasting your time.

**Comment by gakimball** [comment]
Into it, feel free to submit a pull request on the [ZURB Template](https://github.com/zurb/foundation-zurb-template) repository.

Also note that since the Gulpfile is ES6-compatible, you can do this:

```js
gulp.src('src/assets/js/**/*.js')
  // ...
  .pipe(gulp.dest(`${PATHS.dist}/assets/js`));
```

**Comment by phifa** [comment]
@colin-marshall I hope I did it the right way. First time doing this. @gakimball I could not get it to work with ES6. Please change it if that would be better. I am not so much worried about the credits :smile: 
https://github.com/zurb/foundation-zurb-template/pull/33

**Comment by gakimball** [comment]
Merged zurb/foundation-zurb-template#33

Thanks! :)
